db/migrate/20240111000127_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.string :CommentsCounter
+      t.string :LikesCounter
```

Comment:
- [ ] Hello, fantastic progress! 🌟 When working with Rails, it's customary to use snake_case for column names. In the given migration, I recommend changing the column names for "CommentsCounter" and "LikesCounter" to follow this convention:

```suggestion
      t.string :comments_counter
      t.string :likes_counter
```